src/main/res/values/strings.xml

```diff
@@ -85,23 +85,23 @@
   <string name="name">Name</string>
   <string name="network_error_description">Network error.</string>
   <string name="never">Never</string>
-  <string name="new_updates_available">New updates available</string>
+  <string name="new_updates_available">New application versions available</string>
   <plurals name="new_updates_description_format">
     <item quantity="one">%d new update.</item>
-    <item quantity="other">%d new updates.</item>
+    <item quantity="other">%d applications have newer versions available.</item>
   </plurals>
   <string name="no_description_available_description">No description available.</string>
   <string name="no_proxy">No proxy</string>
   <string name="not_updated_description">This repository has not been used yet. You need to enable it to view
     the applications it provides.</string>
   <string name="number_of_applications">Number of applications</string>
   <string name="ok">OK</string>
-  <string name="open_link_confirm_format">Open link %s?</string>
+  <string name="open_link_confirm_format">Open the link \"%s\"?</string>
```

Comment:
(quotation marks, again)

Reply:
If %s is referencing the link, that is correct I should think.

Reply:
I told earlier that I want to avoid quotation marks. Simple "Open the link %s?" is enough.

Reply:
OK, but what is "%s"?

Reply:
URL. "Open the link https://google.com?"

Reply:
Then the quotes are correct.

Reply:
They may be actually _correct_ in many places, which doesn't mean they're _necessary_. As I said, I prefer to avoid quotes.

Reply:
What I meant to say is that omitting them is incorrect. Do you like typographic quotes better “…” ?

Reply:
Why it is not correct? URL is a single line without any spaces within. I don't think using quotes is even good here.

Reply:
Because it is "the" link, pointing to an unreference. "Open link %s" is not a way to get around it.

- Open the link "%s"
or
- Open the link ___ %s.
___ could be "named" "at the address", etc.

Reply:
Maybe "Open the link at %s?" then?